larray/core/array.py

```diff
@@ -547,6 +547,92 @@ def _doc_agg_method(func, by=False, long_name='', action_verb='perform', extra_a
 
 obj_isnan = np.vectorize(lambda x: x != x, otypes=[bool])
 
+
+def equal(a1, a2, rtol=0, atol=0, nan_equals=False):
```

Comment:
I think we should either use equals here or change the equals method to equal. We should have a rule for this (applying s or not) and be consistent. Same thing for nan_equals. We should also have a rule for them too. Not necessarily the same than for function/method names.